### PR TITLE
For EmberTimes No. 89: Add Octane Guides Quest Issue + Nested Anglebracket RFC

### PR DESCRIPTION
**Pull request by jayjayjpg**

### CODE REVIEW

File: source/2019-03-15-the-ember-times-issue-89.md [hunk 1]
```diff
@@ -1,23 +1,34 @@
...
+Your work will contribute to a big initiative the Learning Core team and friends have been working on for a while and which will be highlighted at [EmberConf](https://emberconf.com) next week!
 
+You can read more about how to contribute and what to look out for in the [description of the related Quest Issue](https://github.com/ember-learn/guides-source/issues/588) and thank you for your support - every little bit helps! ⭐️
 
 ---
 
-## [SECTION TITLE](#section-url)
+## [Invoking Components From All the Directories With Angle Brackets 📂](https://github.com/emberjs/rfcs/pull/457)
+
+Even though [angle bracket syntax](https://guides.emberjs.com/release/reference/syntax-conversion-guide/) is the modern and already available way of invoking components in Ember, it is still not possible to invoke all your components in your Ember app this way (yet!).
```
Comment (chrisrng) [inline source/2019-03-15-the-ember-times-issue-89.md]:
```suggestion
Even though the [angle bracket syntax](https://guides.emberjs.com/release/reference/syntax-conversion-guide/) is the modern and already available way of invoking components in Ember, it is still not possible to invoke all your components in your Ember app this way (yet!).
```

File: source/2019-03-15-the-ember-times-issue-89.md [hunk 2]
```diff
@@ -1,23 +1,34 @@
...
 
-<SOME-INTRO-HERE-TO-KEEP-THEM-SUBSCRIBERS-READING>
+Polish up the Guides preview for Octane ⛽️, invoke nested components with angle bracket syntax 📐, ...
 
 ---
 
-## [SECTION TITLE](#section-url)
+## [Copy Editors For the Octane Guides Needed 📝](https://github.com/ember-learn/guides-source/issues/588)
+
+Do you have **a few minutes** to read a page of the [Work in Progress (WIP) of the new Octane Guides](https://github.com/ember-learn/guides-source/issues/588) and send a PR for any kind of **small fix**? It's all about **spelling, typos and factual accuracy** in this round of copy-editing for the Guides and your help - however big or small - is greatly appreciated! 🖤
```
Comment (chrisrng) [inline source/2019-03-15-the-ember-times-issue-89.md]:
```suggestion
Do you have **a few minutes** to read a page of the [Work in Progress (WIP) of the new Octane Guides](https://github.com/ember-learn/guides-source/issues/588) and send a PR for any kind of **small fix**? It's all about **spelling, typos and factual accuracy** in this round of copy-editing for the Octane Guides, and your help - however big or small - is greatly appreciated! 🖤
```

**Review by chrisrng — APPROVED** [review]
LGTM! :)

**Review by MinThaMie — APPROVED** [review]
Looks good :)

**Pull request merged**